cmd/medius/images/push.go

```diff
@@ -64,6 +68,7 @@ func NewPublishImagesCommand(options *common.Options) *cobra.Command {
 	}
 	publishCmd.Flags().BoolVar(&options.PublishImagesOptions.ForceBuild, "force", options.PublishImagesOptions.ForceBuild, "Force a rebuild and push")
 	publishCmd.Flags().IntVar(&options.PublishImagesOptions.Workers, "workers", options.PublishImagesOptions.Workers, "Number of parallel workers")
+	publishCmd.Flags().StringVar(&options.PublishImagesOptions.TargetRegistry, "target-registry", "", "Target registry to push images to")
```

Comment:
@0xFelix could you add a little bit of context why the existing `--registry` can't be used for that?

Reply:
If this is to separate `--source-registry` from `--target-registry` for different subcommands, how about renaming `registry`?

Reply:
I added this flag so it is possible to check the production registry if an update is necessary and to push it to an intermediate registry first (for verification purposes).

Reply:
I would also be fine to remove the `--registry` flag from the base args and just add on-demand to each command a more specific flag which is less ambiguous in source/target scenarios.

Reply:
Done, can you review again?

Reply:
@0xFelix can it be that you added more to this PR than the description says?

Reply:
Oh, you want #21 first. Shouldn't it be the other way round?